addons/binding/org.openhab.binding.konnected/.classpath

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
pardon my ignorance are these all of the .classpath and .project files?

Reply:
All the generated files which where created during your build. I only left the comment here because it is the first file of your PR and I cannot add comments to binary files on github :)

Reply:
will do

Reply:
removed